src/js/charts/histogram.js

```diff
@@ -209,14 +209,14 @@
     this.init(args);
   }
 
-  const defaults = {
-    binned: false,
-    bins: null,
-    processed_x_accessor: 'x',
-    processed_y_accessor: 'y',
-    processed_dx_accessor: 'dx',
-    bar_margin: 1
+  const options = {
+    bar_margin: [1, "number", "the margin between bars"],
```

Comment:
I've changed my mind a bit on the description strings. I'm not sure if including them with metricsgraphics itself is the right approach, as it doesn't really solve any specific problem.

How about we move the API documentation to `examples/` in a new file called api.html, which will dynamically generate a list of textual options in table form:

* The metricsgraphics defaults themselves (import the library)
* A seperate file called "descriptions.js" which contains a mapping of option names to textual descriptions

This could completely replace the wiki, and should be much easier to keep up to date. 

Thoughts?